lib/builder-webpack4/src/preview/iframe-webpack.config.ts

```diff
@@ -124,7 +124,7 @@ export default async ({
     // stats: 'errors-only',
     output: {
       path: path.resolve(process.cwd(), outputDir),
-      filename: '[name].[hash].bundle.js',
+      filename: isProd ? '[name].[contenthash:8].iframe.bundle.js' : '[name].iframe.bundle.js',
```

Comment:
I added `iframe` here and `manager` in manager config, so now chunks from different bundles will be separated and emit to different files.